_install-and-configure/configuring-opensearch/logs.md

```diff
@@ -174,17 +173,46 @@ In this example, OpenSearch logs indexing operations that take 15 seconds or lon
 
 A line from `opensearch_index_indexing_slowlog.log` might look like this:
 
-```
+```plaintext
 node1 | [2019-10-24T19:48:51,012][WARN][i.i.s.index] [node1] [some-index/i86iF5kyTyy-PS8zrdDeAA] took[3.4ms], took_millis[3], type[_doc], id[1], routing[], source[{"title":"Your Name", "Director":"Makoto Shinkai"}]
 ```
 
 Slow logs can consume considerable disk space if you set thresholds or levels too low. Consider enabling them temporarily for troubleshooting or performance tuning. To disable slow logs, return all thresholds to `-1`.
 
+## Search request slow logs
```

Comment:
I would put this section before the old `Slow logs` section, since (IMO) they're way more useful.

I would also rename the existing `Slow logs` section to `Shard slow logs` to clarify that they're emitted at the shard level.

Reply:
Done for both. Also added a sentence in each section to clarify the difference.